### PR TITLE
Type in the gap Webflow

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Marked if applied to all insights:

- [x] Add type in the gap and change context to standalone to all applicable questions

- [x] Change some questions to support type in the gap

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/basics-of-web/element-hierarchy.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/basics-of-web/element-hierarchy.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/basics-of-web/the-element-panel.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/basics-of-web/the-element-panel.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/basics-of-web/webflow-the-box-model.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/basics-of-web/webflow-the-box-model.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/diy/columns.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/diy/columns.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/diy/containers.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/diy/containers.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/diy/divisions.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/diy/divisions.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/diy/getting-started.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/diy/getting-started.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/diy/sections.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/diy/sections.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/intro-components/forms.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/intro-components/forms.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/intro-components/navigation.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/intro-components/navigation.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/intro-components/sliders.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/intro-components/sliders.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/intro-components/symbols.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/intro-components/symbols.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/layouts/webflow-display-settings.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/layouts/webflow-display-settings.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/layouts/webflow-flexbox.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/layouts/webflow-flexbox.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/layouts/webflow-margins.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/layouts/webflow-margins.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/layouts/webflow-overviews.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/layouts/webflow-overviews.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/layouts/webflow-paddings.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/layouts/webflow-paddings.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/style/classes.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/style/classes.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/style/states.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/style/states.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/style/the-style-panel.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/style/the-style-panel.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/style/webflow-html-tags.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/style/webflow-html-tags.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-i/filter-templates.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-i/filter-templates.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-i/quick-start.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-i/quick-start.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-i/template-example.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-i/template-example.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-i/where-to-find-templates.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-i/where-to-find-templates.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-ii/webflow-template-editing-2.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-ii/webflow-template-editing-2.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-ii/webflow-template-editing-3.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/webflow-templates-ii/webflow-template-editing-3.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/what-is-webflow/css.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/what-is-webflow/css.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/what-is-webflow/html.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/what-is-webflow/html.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/what-is-webflow/the-designer.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/what-is-webflow/the-designer.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/what-is-webflow/what-i-need-to-use-webflow.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/what-is-webflow/what-i-need-to-use-webflow.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)
- [ ] [enkidevs/curriculum/nocode-webflow/webflow-introduction/what-is-webflow/what-is-webflow.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/nocode-webflow/webflow-introduction/what-is-webflow/what-is-webflow.md?ref=764fa078460ba032b1d754b0317a7380ee37100b)

File: nocode-webflow/webflow-introduction/diy/divisions.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ category: how-to
 practiceQuestion:
   formats:
     - fill-in-the-gap
-  context: relative
+    - type-in-the-gap
```
Comment (kapnobatai137) [inline nocode-webflow/webflow-introduction/diy/divisions.md:8]:
```suggestion
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**